db/scripts/dump.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS AuctionDB;
+
+CREATE TABLE IF NOT EXISTS `AuctionDB`.`USERS` (
+`ID` BIGINT NOT NULL AUTO_INCREMENT,
+`NAME` VARCHAR(255) NULL,
```

Comment:
As for me it is more clear to call this field as 'LOGIN', because name sounds like some value object where contains first name, last name etc. And I'm not sure is it a good idea to allow user make login for 255 symbols :)